CONTRIBUTING.md

```diff
@@ -1,6 +1,38 @@
 # Contributing
 First of all, thank you for contributing. It’s appreciated. This guide details how to use issues and pull requests to improve this project.
 
+Table of Contents
+=================
+
+  * [Requirements:](#requirements)
+  * [Installation](#installation)
+  * [Running](#running)
+  * [Making changes](#making-changes)
+    * [Branching](#branching)
+    * [Commit message](#commit-message)
+  * [Creating an Issue](#creating-an-issue)
+  * [Tests](#tests)
+
+## Requirements:
+* NodeJS(v4 LTS)
+* git crypt
+
+## Installation
+Clone this repository
+
+```bash
+$ git clone https://github.com/sphereio/sphere-product-type-import.git
+```
+
+We are using git-crypt for encrypting configs.
+Run
+
+```bash
+$ git-crypt unlock
+```
+
+to unlock the config file.
```

Comment:
Yeah, and actually a better solution will be to remove the hardcoded project keys in the test. So that if anybody wants to test, they will fetch their development project credentials from admin.sphere.io. Also have proper setup and teardown methods.

With that we don't even need to have the credentials in the project at all. One credential will work for all the tests.